dashboard/api.py

```diff
@@ -510,6 +511,23 @@ def get_future_exam_runs(course):
             order_by('date_first_schedulable').values_list('date_first_schedulable', flat=True))
 
 
+def get_past_recent_exam_run(course):
+    """
+    Return a recent exam run
+
+    Args:
+        course (courses.models.Course): A course
+
+    Returns:
+        tuple: a list of dates when future exams become schedulable
```

Comment:
it is returning a date range i.e start and end date of a last schedulable exam run? Am i right. Can we make it more clear?